src/constants/index.ts

```diff
@@ -238,9 +287,26 @@ export const D4_SWAP_TOKEN = new Token(
   "saddled4",
   "Saddle alUSD/FEI/FRAX/LUSD",
   saddleLogo,
+  false,
+  true,
 )
 
 // Stablecoins
+const SUSD_CONTRACT_ADDRESSES: { [chainId in ChainId]: string } = {
+  [ChainId.MAINNET]: "0x57Ab1ec28D129707052df4dF418D58a2D46d5f51",
+  [ChainId.ROPSTEN]: "",
+  [ChainId.HARDHAT]: "0x0E801D84Fa97b50751Dbf25036d067dCf18858bF",
+}
+export const SUSD = new Token(
+  SUSD_CONTRACT_ADDRESSES,
+  18,
+  "sUSD",
+  "nusd",
```

Comment:
s/nusd/susd

https://www.coingecko.com/en/coins/susd

Reply:
The id required for API call is still nusd :S